FrontEnd/src/components/HeaderFooter/header/navbar/Profile.module.css

```diff
@@ -1,36 +1,90 @@
 .header-profile-section {
-    display: flex;
-    align-items: center;
-    gap: 6px;
+  display: flex;
+  align-items: center;
+  gap: 6px;
 }
 
 .header-profile__avatar {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-end;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
 
-    border-radius: 100px;
-    border: 1px solid #E2E5EB;
-    background: #FFF;
+  border-radius: 100px;
+  border: 1px solid #e2e5eb;
+  background: #fff;
```

Comment:
Do we have css variables? I would recommend to use them

Reply:
Done